src/safe_apps/tests/test_models.py

```diff
@@ -70,3 +71,12 @@ class FeatureTestCase(TestCase):
     def test_str_method_outputs_feature_key(self) -> None:
         feature = FeatureFactory.create()
         self.assertEqual(str(feature), f"Safe App Feature: {feature.key}")
+
+
+class SocialProfileTestCase(TestCase):
+    def test_str_method_outputs_social_profile_name(self) -> None:
```

Comment:
We should also test what happens when you save a model with invalid `TextChoices`.

And also that a Social Profile cannot exist without a `SafeApp`.